spec/models/shipment_spec.rb

```diff
@@ -1,5 +1,84 @@
 require "rails_helper"
 
 RSpec.describe Shipment, type: :model do
-  # write your tests here
+  describe "" do
+
+
+    let(:valid_shipment) {
+      Shipment.new(
+        company: Company.new(name: "New Co"),
+        origin_country: "AI",
+        destination_country: "ER",
+        tracking_number: "UM459056399US",
+        slug: "slug"
+      )
```

Comment:
It's a good catch, but it might be complex when working with test object and managing them on large project afterwards, you can take a look at how to use https://github.com/thoughtbot/factory_bot, which is quite common in Rails project, or even a lighter one with mini-test `fixtures` that was built-in with Rails